lib/figgy/configuration.rb

```diff
@@ -40,14 +38,45 @@ def initialize
       define_handler 'json' do |contents|
         JSON.parse(contents)
       end
+
+      @overlays = [Overlay.new('root', nil, @roots)]
     end
 
+    # Sets the +root+ to the given directory, or to a Vault client.
+    #
+    # If just a path is given, it sets the root to that file directory
+    # path. If a path and Vault client are given, it sets the root to
+    # a Vault client rooted at the given path.
+    def set_root(path, vault = nil)
+      new_root = if vault
+                   VaultRoot.new(vault, path)
+                 else
+                   FileRoot.new(File, File.expand_path(path))
+                 end
+
+      @roots = [new_root]
+      @overlays = [Overlay.new('root', nil, @roots)]
+    end
```

Comment:
Did you decide not to do two methods here?

Reply:
nah i just forgot 😆 fixing it now